app/controllers/cohorts_controller.rb

```diff
@@ -20,11 +20,26 @@ def show
     @cohort = cohort_with_preloads.first
     
     if params[:inactive].present?
-      @cohort_clients = @cohort.cohort_clients.joins(:client)
-    else
-      @cohort_clients = @cohort.cohort_clients.joins(:client).where(active: true)
+      case params[:population]&.to_sym
+        when :housed
+          @cohort_clients = @cohort.cohort_clients.joins(:client).where.not(housed_date: nil).where(ineligible: [nil, false])
+        when :active
+          @cohort_clients = @cohort.cohort_clients.joins(:client).where(housed_date: nil, ineligible: [nil, false])
+        when :ineligible
+          @cohort_clients = @cohort.cohort_clients.joins(:client).where(ineligible: true)
+      end
+    else    
+      case params[:population]&.to_sym
```

Comment:
@everestwitman I know I wrote this, but I'm thinking we should throw the case statement in a method in a concern.  It should take a scope and population.  Returns a scope.  I believe there are 4 instances of it, so that should make it a bit easier to adjust/maintain.